### DISCUSSION TITLE
analogWrite() and DAC

### DISCUSSION THREAD

**Discussion by ikkei3** [SpenceKonde/DxCore#409] [Q&A]
1.5.5 has been released.
I have AVR64DB32 and AVR64DD32.
analogWrite() works on AVR64DB32 but not on AVR64DD32.
DAC does not work on AVR64DB32 and AVR64DD32.
do i need to configure anything?

**Comment by SpenceKonde** [comment]
<--- issue #409

**Comment by SpenceKonde** [comment]
This issue was solved a  few versions ago,
